packages/devtools/lib/src/ui/dragscroll.dart

```diff
@@ -0,0 +1,94 @@
+import 'dart:html';
+
+class DragScroll {
+  var mousemove = 'mousemove';
+  var mouseup = 'mouseup';
+  var mousedown = 'mousedown';
+  var EventListener = 'EventListener';
+
+// ??? what are these ???
+//  var addEventListener = 'add'+EventListener;
+//  var removeEventListener = 'remove'+EventListener;
+
+
+
+  num newScrollX;
+  num newScrollY;
+
+  var dragged = [];
+
+  void reset(int i, /*type?*/ eventListener) {
+    while( i < dragged.length) {
+      eventListener = dragged[i++];
+      eventListener = eventListener.container ?? eventListener;
+      /* what are mu mm md? mouse up, mouse move, and mouse down? */
+      eventListener.removeEventListener(mousedown, eventListener.md, false);
+      window.removeEventListener(mouseup, eventListener.mu, false);
+      window.removeEventListener(mousemove, eventListener.mm, false);
+    }
+
+    dragged = document.getElementsByClassName('dragscroll');
```

Comment:
document.getElementsByClassName returns a List<Node>. So List<Node> must be the type of dragged?